contrib/get_kind.sh

```diff
@@ -2,4 +2,5 @@
 
 echo "go get -u sigs.k8s.io/kind"
 
-go get -u sigs.k8s.io/kind
+export GO111MODULE="on"
+go get -u sigs.k8s.io/kind@master
```

Comment:
Given the comments we had from the maintainers do you think that is a good idea to use HEAD/master? Is there a more suitable version that we can pin to or not?

Reply:
I just updated it with the current master sha, since we know that works right now: kubernetes-sigs/kind@0a6ceae